Mathlib/Topology/Defs/Filter.lean

```diff
@@ -202,8 +202,8 @@ infixl:300 " ⤳ " => Specializes
 equivalent properties hold:
 
 - `𝓝 x = 𝓝 y`; we use this property as the definition;
-- for any open set `s`, `x ∈ s ↔ y ∈ s`, see `inseparable_iff_open`;
-- for any closed set `s`, `x ∈ s ↔ y ∈ s`, see `inseparable_iff_closed`;
+- for any open set `s`, `x ∈ s ↔ y ∈ s`, see `Inseparable.mem_open_iff`;
```

Comment:
```suggestion
- for any open set `s`, `x ∈ s ↔ y ∈ s`, see `inseparable_iff_forall_open`;
```
We're claiming to point at the equivalence to inseparable, don't we?

Reply:
Yes, indeed! Fixed, also for closedness.